### DISCUSSION TITLE
Custom Tabs?

### DISCUSSION THREAD

**Discussion by sarasensible** [coroot/coroot#211] [General]
I would love to use coroot as a single pane of glass for our SLOs and KPIs.  However there are some views I would like to add to answer additional questions we have about the data.

Is it possible to add custom tabs to the dashboard (potentially sourced from Grafana)?  Or is coroot extensible enough to permit this?

**Comment by def** [comment]
Yes, Coroot now supports custom dashboards: https://docs.coroot.com/dashboards/overview
